corehq/apps/auditcare/utils/export.py

```diff
@@ -164,7 +166,18 @@ def get_sql_start_date():
     all auditcare data in Couch will be obsolete and/or archived before
     SQL data. It should be removed when the data in Couch is no longer
     relevant.
+
+    NOTE the output is being hardcoded for the time historical auditcare events are copied to SQL
     """
+    if settings.SERVER_ENVIRONMENT == 'production':
```

Comment:
@millerdev  @orangejenny  I had another approach for it to have a Django setting for each environment with timestamp as values and use that here.  But that would have involved changes in commcare cloud. Since this is a temporary change and would be removed in some time so I thought it would make sense to keep stuff in one place so that it would be easy to remove.

Reply:
I think this is fine, agree that simplicity is good.